README.md

```diff
@@ -152,7 +152,7 @@ Will generate an output folder such as:
 
 Where both schemas have been rendered through their corresponding templates.
 
-**N.B.** It is not necessary to specify a [file attribute](#user-content-file) within the templates object if mapping a folder like this, however, if there are schemas that do not map directly to template files, i.e. have the same name, Matic will attempt to use the file specified by the file attribute as a default template.
+**N.B.** It is not necessary to specify a [file attribute](#user-content-file) within the templates object if mapping a folder like this. However, if there are schemas that do not map directly to template files—i.e., have the same name—Matic will attempt to use the file specified by the file attribute as a default template.
```

Comment:
"—i.e., have the same name—"
Looks odd to me. Maybe just:
"(have the same name)"